ccgains/trades.py

```diff
@@ -257,9 +267,43 @@ def _parse_trade(str_list, param_locs, default_timezone):
         else:
             pdict[key] = val
 
+    # Get missing fees if requested
+    if fill_missing_fees and pdict['kind'] == 'Withdrawal':
+        tx = pdict['mark']
+        fee_curr = pdict['fee_currency']
```

Comment:
If there is no `fee_amount`, `fee_currency` might be empty. Rather use `sell_currency` here.